R/tbl_survfit_quantiles.R

```diff
@@ -0,0 +1,245 @@
+#' Survival Quantiles
+#'
+#' Create a gtsummary table with Kaplan-Meier estimated survival quantiles.
+#'
+#' @param data (`data.frame`)\cr
+#'   A data frame
+#' @param y (`string`)\cr
+#'   A string with the survival outcome, e.g. `survival::Surv(time, status)`.
```

Comment:
why not this (with default):
(Surv or string)
an object of class Surv created using survival::Surv(). This object will be passed as the left-hand side of the formula constructed and passed to survival::survfit(). This object can also be passed as a string. 

from `?cardx::ard_survival_survfit`

Reply:
Do you mean why not allow for both quoted and unquoted inputs?

There is a function in cardx that handles the inputs of either string or unquoted code, and I was just being lazy and didn't copy a version of that function here. Do you think we should update to allow both?

Reply:
@ddsjoberg will add functionality for both string and expression input

Reply:
This has been added.